src/core/containers/filter.jsx

```diff
@@ -29,11 +29,11 @@ export default class FilterContainer extends React.Component {
 
     return (
       <div>
-        {filter === null || filter === false || filter === "false" ? null :
+        {filter === null || filter === false ? null :
```

Comment:
Looking at this, perhaps we could simply cast `null` filter to `false`?

Reply:
We'll change default for `filter` to `false`